articles/azure-app-configuration/faq.yml

```diff
@@ -130,11 +130,14 @@ sections:
           You'll receive an HTTP status code 429 response under these circumstances:
           
           * Exceeding the daily request limit for a store in the Free tier.
-          * Temporary throttling due to a high request rate for a store in the Standard tier.
+          * Exceeding the hourly request limit for a store in the standard tier.
+          * Momentary throttling due to a large burst of requests.
           * Excessive bandwidth usage.
           * Attempting to create or modify a key when the storage quote is exceeded.
           
           Check the body of the 429 response for the specific reason why the request failed.
+          
+          A configuration store may experience momentary throttling if it receives a large burst of requests. App Configuration clients, such as the Azure SDK, configuration provider libraries, and Azure Pipeline tasks, automatically retry on throttled requests. For any applications using one of these clients, or a custom client that retries on throttled requests, this momentary throttling should go unnoticed, should it occur.
```

Comment:
I wonder if it should be right under the bullet where momentary throttling is mentioned.

Reply:
Do you mean putting this in front of the "Check the body..."?

Reply:
No I meant bulleted list with possible explanation.

          * Exceeding the daily request limit for a store in the Free tier.
          * Exceeding the hourly request limit for a store in the standard tier.
          * Momentary throttling due to a large burst of requests.

              A configuration store may experience momentary throttling if it receives a large burst of requests. App Configuration clients, such as the Azure SDK, configuration provider libraries, and Azure Pipeline tasks, automatically retry on throttled requests. For any applications using one of these clients, or a custom client that retries on throttled requests, this momentary throttling should go unnoticed, should it occur.

          * Excessive bandwidth usage.
          * Attempting to create or modify a key when the storage quota is exceeded.



But perhaps it isn't worth it.

Reply:
Alternatively, we can do

* ...
* Momentary throttling due to a large burst of requests*.
* ...

Check the body of the 429 response for the specific reason why the request failed.

\*A configuration store may experience momentary throttling ...

Reply:
Not a bad idea.

Reply:
Cool. Updated.